code/modules/surgery/amputation.dm

```diff
@@ -30,6 +30,7 @@
 		/obj/item/shears = 300,
 		TOOL_SCALPEL = 100,
 		TOOL_SAW = 100,
+		/obj/item/shovel/serrated = 75,
```

Comment:
Could always make this literally any shovel. Kind of funnier if so.

Reply:
Nahh